fp-bindgen/src/types/type_ident.rs

```diff
@@ -267,13 +268,6 @@ fn path_to_string(path: &syn::Path) -> String {
         .join("::")
 }
 
-// Used to remove the 'Serializable' bound from generated types, since this trait only exists in fp-bindgen
-// and doesn't exist at runtime.
-fn is_runtime_bound(bound: &str) -> bool {
```

Comment:
Ah, this wasn't used anymore? Good, since it was a bit hacky :)

Reply:
Well, there were two copies of this function. So the hackiness has been reduced from 2 to 1 😅